farcy/__init__.py

```diff
@@ -246,7 +250,9 @@ def handle_pr(self, pr):
         issue_count = 0
         did_work = True
         exception = False
-        existing_comments = pr.review_comments()
+        existing_comments = list(filter_comments_from_farcy(
```

Comment:
It was a [building block](https://github.com/appfolio/farcy/blob/master/farcy/helpers.py#L40) of one of the other helpers. [It is even tested.](https://github.com/appfolio/farcy/blob/master/test/test_helpers.py#L31)